projects/Lorenzo/index.js

```diff
@@ -0,0 +1,23 @@
+const { sumTokensExport } = require("../helper/sumTokens");
+
+const LorenzoOwners = [
+    "bc1pzd6luyardlle9f7lul2y8fl72c22p6vxspc4k4g4gzgjf8975s0sr042yt",
```

Comment:
what are these addresses?

Reply:
These address are mpc custody address support by cobo. Used to receive BTC staked by users, and then mint lorenzo native gas fee token stBTC to user.

Reply:
but there is no deposit atm?

Reply:
Already have users to stake and get lorenzo native fee token(stBTC)